src/app/features/fss-search/fss-search.component.ts

```diff
@@ -247,4 +253,42 @@ export class FssSearchComponent implements OnInit {
     this.eventAdvancedSearchTokenRefresh.next();
   }
 
+  transformSearchAttributesToFilter(attributeSearchResult: any) {
```

Comment:
`attributeSearchResult` is treated as an array later in the code so the type should probably be `any[]`

also the name should be pluralised to `attributeSearchResults`

Reply:
updated as suggested. Thanks